src/fiskaltrust.ifPOS/v1/de/Models/StartTransactionResponse.cs

```diff
@@ -6,31 +6,19 @@ namespace fiskaltrust.ifPOS.v1.de
     [DataContract]
     public class StartTransactionResponse
     {
-        [DataMember(Order = 1)]
-        public long TransactionNumber { get; set; }
+        [DataMember(Order = 10)]
+        public ulong TransactionNumber { get; set; }
 
-        [DataMember(Order = 2)]
-        public DateTime StartTime { get; set; }
+        [DataMember(Order = 20)]
+        public DateTime TimeStamp { get; set; }
 
-        [DataMember(Order = 3)]
-        public string CertificateSerialNumber { get; set; }
+        [DataMember(Order = 30)]
+        public string TseSerialNumberOctet { get; set; }
 
-        [DataMember(Order = 4)]
+        [DataMember(Order = 40)]
         public string ClientId { get; set; }
 
-        [DataMember(Order = 5)]
-        public string ProcessType { get; set; }
```

Comment:
Aren´t we currently sending these fields back as signatures after a explicit call to the IPOS?

Reply:
i'm not sure if i'm on the right track.
the ProcessType and ProcessDataBase64 we have in each of the requests, this are used to throw them back to calling queue-code.